lib/message.js

```diff
@@ -20,20 +20,20 @@ const { camelCase, get, invoke } = require('lodash');
 
 const paths = {
   browser: ({ suite }) => ['suite', 'runner', suite.cid, 'browserName'],
```

Comment:
I think this might need an update as well. I did a console.log(ctx) to see what object properties are available - you can see the result here
https://gist.github.com/dylanlive/f508a8201f36353954d7e3b03800b297#file-ctx_object-js-L39

Based on that, this works (at least for chrome):
`browser: () => 'reporter.browser.capabilities.browserName',`